src/assets/css/_images.scss

```diff
@@ -456,6 +456,10 @@
   @include mixins.provider-image('/assets/images/clouds/eks.svg', 130px, 30px);
 }
 
+.km-provider-logo-custom-externalcluster {
+  @include mixins.provider-image('/assets/images/clouds/customkubernetescluster.svg', 130px, 30px);
```

Comment:
can we rename to?
- km-provider-logo-custom
- - km-provider-logo-bringyourown (BRO) means customs